cli/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ _Released 10/25/2023 (PENDING)_
 
 **Bugfixes:**
 
+- Fixed a performance regression with proxy correlation timeouts and requests issued from service workers. Fixed an issue where proxy correlation would leak over from a previous spec. Fixed an issue where redirects of requests that knowingly don't have CDP traffic should also be assumed to not have CDP traffic. Fixes [#28054](https://github.com/cypress-io/cypress/issues/28054) and [#28056](https://github.com/cypress-io/cypress/issues/28056).
```

Comment:
I feel like these should be split up into 3 different bullet points.

Reply:
I don't think the changelog parser will like that but we'll see.

Reply:
Ah I can make it work

Reply:
[`281f61c` (#28060)](https://github.com/cypress-io/cypress/pull/28060/commits/281f61c40f7baa1c098c8b33c5c18059188d30c9)